index.js

```diff
@@ -19,11 +19,13 @@ export class VectorTileFeature {
         this.properties = {};
 
         this.extent = extent;
+        /** @type {0 | 1 | 2 | 3} */
         this.type = 0;
 
         /** @type {number | undefined} */
         this.id = undefined;
 
+        // Private
```

Comment:
I'm not sure this will remove the following fields from the generated types unfotunately...
I'm not familiar enough with typescript generation from javascript comments.
What I wanted to achieve is the the "_" properties will not be present in the index.d.ts file...